client/src/views/client-area/statistics/CustomOrderStatisticsPage.vue

```diff
@@ -0,0 +1,719 @@
+              />
+            </div>
+          </div>
+          <div class="storage-data-consumption">
+            <ion-skeleton-text
+              :animated="true"
+              class="skeleton-loading-text"
+            />
+            <div class="storage-data-consumption-content">
+              <div class="consumption-item">
+                <div
+                  id="firstBar"
+                  class="consumption"
+                >
+                  <div class="consumption-number">
+                    <ion-skeleton-text
+                      :animated="true"
+                      class="skeleton-loading-text"
+                    />
+                    <ion-skeleton-text
+                      :animated="true"
+                      class="skeleton-loading-text"
+                    />
+                  </div>
+                  <ion-skeleton-text
+                    :animated="true"
+                    class="skeleton-loading-text"
+                  />
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </template>
+    <template v-else-if="!querying && !organizationStats && !error">
+      {{ $msTranslate('clientArea.statistics.noStats') }}
+    </template>
+    <ion-text
+      class="statistics-error body"
+      v-show="error"
+    >
+      {{ $msTranslate(error) }}
+    </ion-text>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { BmsAccessInstance, DataType, BmsOrganization, OrganizationStatsResultData, CustomOrderDetailsResultData } from '@/services/bms';
+import { onMounted, ref, computed } from 'vue';
+import { IonCard, IonText, IonIcon, IonTitle, IonSkeletonText } from '@ionic/vue';
+import { formatFileSize } from '@/common/file';
+import { person, star, pencil, arrowForward } from 'ionicons/icons';
+import { isDefaultOrganization } from '@/views/client-area/types';
+import { MsReportTheme, MsReportText } from 'megashark-lib';
+
+const props = defineProps<{
+  organization: BmsOrganization;
+}>();
+
+const organizationStats = ref<OrganizationStatsResultData | undefined>(undefined);
+const contractDetails = ref<CustomOrderDetailsResultData | undefined>(undefined);
+const useStoragePercentage = ref<number>(0);
+const useStorageWidth = computed(() => `${getUseStoragePercentage()}%`);
+const totalData = ref<number>(0);
+const querying = ref(true);
+const error = ref('');
+
+function getStorageSize(): number {
+  if (!contractDetails.value) {
+    return 0;
+  }
+  // Slice of 100Go
+  return contractDetails.value.storage.quantityOrdered * 100 * 1024 * 1024 * 1024;
+}
+
+function getRemainingStorageSize(): number {
+  if (!contractDetails.value) {
+    return 0;
+  }
+  return getStorageSize() - totalData.value;
+}
+
+function getUseStoragePercentage(): number {
+  if (!organizationStats.value) {
+    return 0;
+  }
+  const ordered = getStorageSize();
+  const current = totalData.value;
+
+  if (current / ordered >= 1) {
```

Comment:
You can replace this if/else with
`return Math.min(Math.round((current / ordered) * 100), 100)`.

`round` makes sure that if your percentage is `13.333333333` it gets rounded to `13`.
`min()` takes the lesser values of the two: if your percentage goes above 100, it will pick the second parameter (100) because it's the lowest of the two.